aclocal.m4

```diff
@@ -43,6 +43,5 @@ m4_include([tool/m4/ruby_stack_grow_direction.m4])
 m4_include([tool/m4/ruby_try_cflags.m4])
 m4_include([tool/m4/ruby_try_cxxflags.m4])
 m4_include([tool/m4/ruby_try_ldflags.m4])
-m4_include([tool/m4/ruby_type_attribute.m4])
```

Comment:
Don't mix other change

Reply:
unexpected change I ll roll it back

Reply:
This is not fixed yet

Reply:
sorry I forgot in the meantime

Reply:
Thank you for update! I'll merge this after CI passes.